javalib/src/main/scala/java/time/temporal/TemporalAmount.scala

```diff
@@ -0,0 +1,3 @@
+package java.time.temporal
+
+trait TemporalAmount {}
```

Comment:
And here we go down the rabbit hole!.

I suggest a comment that this is an implementation stub, just enough to get Duration.scala to work.
That would yield `least astonishment` to future readers of the file.

Reply:
Added commented out methods for readability. This was only added to complete the `class` signature of `Duration`.

Reply:
I understand completely... In six months time, somebody, perhaps you or I, will be singing your praises for the clue.
In fact, I will even sing a verse of praise now for putting up with the tedium ;-)